main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     cat
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat eats
+cat.eat();
+//  action 3: the cat pounces
+cat.pounce();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     bus
+//  action 1: the bus stops
+bus.stop();
+//  action 2: the bus turns left
+bus.turnLeft();
+//  action 3: the bus honks horn
+bus.honkHorn();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:      grass
+//  action 1:  the grass photosynthesizes
+grass.photosynthesize();
+//  action 2:  the grass disperses seed
+grass.disperseSeed();
+//  action 3:  the grass dies
+grass.die();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     bell
+//  action 1: the bell makes a ringing sound
+bell.ring();
+//  action 2: the bell cracks
+bell.crack();
+//  action 3: the bell swings
+bell.swing();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: toe
+//  action 1: the toe is stubbed
+toe.stub();
+//  action 2: the toe wiggles up and down
+toe.wiggleUpandDown();
+//  action 3: the toe grows toenail
+toe.growToenail();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:     hand
+//  action 1: the hand waves
+hand.wave();
+//  action 2: the hand gives a thumbs up
+hand.thumbsUp();
+//  action 3: the hand gives a thumbs down
+hand.thumbsDown();

```

Comment:
`give` is the action.  Although, a better word would be 'forms'.  i.e. the hand forms a thumbs up
either way, the pseudo-code is missing an action word.
- [ ] revise